opm/parser/eclipse/IntegrationTests/IOConfigIntegrationTest.cpp

```diff
@@ -298,12 +298,7 @@ BOOST_AUTO_TEST_CASE( NorneResttartConfig ) {
     rptConfig.push_back( std::make_tuple(240 , true , boost::gregorian::date( 2006,10,1)) );
     rptConfig.push_back( std::make_tuple(241 , true , boost::gregorian::date( 2006,10,10)) );
 
-
-    ParseContext parseContext;
-    ParserPtr parser(new Parser());
-    DeckConstPtr deck = parser->parseFile("testdata/integration_tests/IOConfig/RPTRST_DECK.DATA", parseContext);
-    EclipseState state( deck , parseContext );
-
+    auto state = Parser::parse("testdata/integration_tests/IOConfig/RPTRST_DECK.DATA");
```

Comment:
I don't like the name of the method (`parse()`): IMO this name applies to parsing in-memory data as equally as to parsing files, network resources and god knows what. what's wrong with `parseFile()`, `parseString()`, etc?